model.py

```diff
@@ -404,6 +404,8 @@ class Patron(Base):
     loans = relationship('Loan', backref='patron')
     holds = relationship('Hold', backref='patron')
 
+    annotations = relationship('Annotation', backref='patron')
```

Comment:
Does it make sense to add an `order_by` argument here to make sure we get most recent annotation first? Or should that go into the circ manager side where the list is actually botained?